crates/sui-core/src/authority/shared_object_version_manager.rs

```diff
@@ -121,6 +121,100 @@ impl SharedObjVerManager {
         }
         Ok(assigned_versions)
     }
+
+    pub fn assign_versions_for_certificate(
```

Comment:
can you undo the code move here so I can see the diffs (if any)? you can add a new impl block at the old location to avoid the mode

Reply:
oh, if you say they are identical i believe you, you can undo the move 🗡️

Reply:
done